lib/order.rb

```diff
@@ -0,0 +1,57 @@
+class Order
+
+  require_relative 'menu'
+  require_relative 'send_sms'
+
+  attr_reader :cart
+
+  include SendTexts
+
+  def initialize(takeaway)
```

Comment:
To me it's not clear what the takeway parameter is that being passed into your order at initialization